Makefile

```diff
@@ -442,6 +443,23 @@ hex_factory: firmware_factory settings_factory
 flash_factory: erase hex_factory
 	nrfjprog -f nrf52 --program $(OUTPUT_DIRECTORY)/full_firmware_factory.hex --sectorerase --verify --reset
 
+#
+# Validation commands
+#
+
+.PHONY: hex_validation
+hex_validation: firmware_release settings_release
+	mergehex -m $(OUTPUT_DIRECTORY)/firmware.hex $(OUTPUT_DIRECTORY)/firmware_settings.hex $(SOFTDEVICE_HEX_PATHNAME) $(BOOTLOADER_HEX_PATHNAME) -o $(OUTPUT_DIRECTORY)/full_firmware.hex
```

Comment:
Can you make `hex_validation` call `hex_release` to build `full_firmware.hex`?

Reply:
Sure, I'll do that right now!

Reply:
Pushed!